packages/backend/src/account/service.ts

```diff
@@ -4,6 +4,7 @@ import { BaseService } from '../shared/baseService'
 export interface AccountService {
   get(id: string): Promise<Account>
   create(scale: number, currency: string): Promise<Account>
+  createSubAccount(parentAccountId: string): Promise<Account>
```

Comment:
Lets change all references of `parentAccountId` to `superAccountId` as per https://www.notion.so/coil/Rafiki-Account-Service-API-6d172588dd534becacfa21e178cd7407#a850539758e445b58fca7a3305249506

Reply:
Why is `super` preferable to `parent`? I find the latter much more straightforward.

Reply:
Never mind; Brandon answered my question.